app/src/main/java/com/heapixLearn/discovery/MainActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.heapixLearn.discovery;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.Toast;
+
+import com.heapixLearn.discovery.Server.Contacts.ServerContact;
+import com.heapixLearn.discovery.Server.Contacts.ServerContactManager;
+import com.heapixLearn.discovery.Server.TypeOfServerError;
+
+import java.util.List;
+
+
+public class MainActivity extends AppCompatActivity {
```

Comment:
@Agent00krosh activities are out of the context of the task, shouldn't be in the pr